input/pagecontent/index.md

```diff
@@ -1,48 +1,45 @@
+
+{% include dependency-table.xhtml %}
+
 ### How to Read this Guide
 
 This guide is divided into several pages which are listed at the top of each page in the menu bar.
 
 - [Home](index.html): This page provides the introduction and scope for this guide.
-- Guidance:
-  - [Guidance](guidance.html): This page provides guidance on using the profiles defined in this guide.
+- [Guidance](guidance.html):  These pages list the guidance for this guide.
+  - [General Guidance](generalguidance.html): This page provides guidance on using the profiles defined in this guide.
   - [Relationship with other IGs](relationship.html): This page provides guidance on the relationship between AU Base and other implementation guides.
   - [Future of AU Base](future.html): This page outlines the yearly update cycle and approach to maturing AU Base.
 - [FHIR Artefacts](artifacts.html): These pages provide detailed descriptions and formal definitions for all the FHIR artefacts defined in this guide.
-  - [Profiles and Extensions](profiles-and-extensions.html): This set of pages describes the profiles and extensions that are defined in this guide to represent Australian local concepts using FHIR. Each profile page includes a narrative description, guidance, and formal definition. Although the guidance typically focuses on the profiled elements and seeks to provide a ‘how-to’ guide when representing concepts, it may also may focus on un-profiled elements to aid with implementation.
-  - [Search Parameters](search-parameters.html): This set of pages lists the search parameters defined in this guide for use in AU operations.
-  - [Terminology](terminology.html): This set of pages lists the value sets and code systems defined in this guide.
+  - [Profiles and Extensions](profiles-and-extensions.html): This page describes the profiles and extensions that are defined in this guide to represent Australian local concepts using FHIR. Each profile page includes a narrative description, guidance, and formal definition. Although the guidance typically focuses on the profiled elements and seeks to provide a ‘how-to’ guide when representing concepts, it may also may focus on un-profiled elements to aid with implementation.
+  - [Search Parameters](search-parameters.html): This page lists the search parameters defined in this guide for use in AU operations.
+  - [Terminology](terminology.html): This page lists the value sets and code systems defined in this guide.
 - [Examples](examples.html): This page lists all the examples used in this guide.
-- [Downloads](downloads.html): This page provides links to downloadable artefacts including the AU Base FHIR NPM package.
+- [Support](support.html): These pages provide supporting material for implementation of AU Base.
+  - [Downloads](downloads.html): This page provides links to downloadable artefacts.
+  - [License and Legal](license.html): This page outlines license and legal considerations relating to FHIR Implementation Guides.
```

Comment:
License and Legal should be "AU Base" not "FHIR Implementation Guides:

Reply:
Fixed, thanks.